ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py

```diff
@@ -161,7 +161,7 @@
   description: Returns the information collected from the HPE Nimble Storage array
   returned: always
   type: complex
-   "nimble_info": {
+  sample: {
```

Comment:
changed the string to "sample" to support ansible-doc command to generate the example in return coloumn